internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -300,12 +300,9 @@ func UpgradeRefreshFunc(ctx context.Context, name, projectID string, client Clus
 	}
 }
 
-func ResourceClusterListAdvancedRefreshFunc(ctx context.Context, projectID string, client ClusterService) retry.StateRefreshFunc {
+func ResourceClusterListAdvancedRefreshFunc(ctx context.Context, projectID string, client admin.ClustersApi) retry.StateRefreshFunc {
```

Comment:
better to change client name to clustersApi or similar

Reply:
Done ✔️